src/features/admin/routes/tab-panel.tsx

```diff
@@ -0,0 +1,29 @@
+import {Box, Typography} from "@mui/material";
+import React from "react";
+
+function TabPanel(props: TabPanelProps) {
+    const {children, value, index, ...other} = props;
+
+    interface TabPanelProps {
```

Comment:
Please move them outside the component. It's fine to use `interface` but the rest is using `type`. Maybe we can align it.

Reply:
done